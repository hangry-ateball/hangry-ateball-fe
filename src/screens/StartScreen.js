import React from 'react'
import { StyleSheet, View, Button, Image } from 'react-native'

const StartScreen = ({ navigation }) => {
  return (
    <View style={styles.container}>
      <Image style={styles.eightBall} source={require('../../assets/ate-ball-home.png')}/>
      <View style={styles.startBtn}>
        <Button
          title="Get Shakin'"
          onPress={() => navigation.navigate('Form')}
        />
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    // paddingTop: 50,
    alignItems: 'center',
    width: '100%',
  },
  startBtn: {
    width: '80%',
  },  
  eightBall: {
    width: 350,
    resizeMode: 'contain'
  }
});


export default StartScreen;