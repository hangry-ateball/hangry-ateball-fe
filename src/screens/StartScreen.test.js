import React from 'react';
import {shallow} from 'enzyme';
import StartScreen from './StartScreen';
describe('StartScreen', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
           const wrapper = shallow(<StartScreen/>);
            expect(wrapper).toMatchSnapshot();
        });
    });
});