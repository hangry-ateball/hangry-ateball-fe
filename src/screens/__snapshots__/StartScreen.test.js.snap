// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartScreen Rendering should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/ate-ball-home.png",
      }
    }
    style={
      Object {
        "resizeMode": "contain",
        "width": 350,
      }
    }
  />
  <View
    style={
      Object {
        "width": "80%",
      }
    }
  >
    <Button
      onPress={[Function]}
      title="Get Shakin'"
    />
  </View>
</View>
`;
